### PR TITLE
Feature mes 1127 journal refresh

### PR DESCRIPTION
**Pull request by inhand**
## Description and relevant Jira numbers

Allow the journal page to be refreshed by a pull to refresh and a refresh button on the page

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [x] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [x] Any new reusable component/widget added to component library on Confluence

- [x] Screenshot(s) captured on the physical device (if applicable)
- [x] Tested by QA
- [ ] PO's approval

### CODE REVIEW

File: src/pages/journal/journal.html [hunk 1]
```diff
@@ -8,13 +8,24 @@
 </ion-header>
 
 <ion-content padding>
+  <ion-refresher (ionRefresh)="pullRefreshJournal($event)">
```
Comment (johnmorleybjss) [inline src/pages/journal/journal.html]: Having tested on the iPad, the distance pulled down to trigger the refresh seems quite small. Could you perhaps set `pullMin` and `pullMax` to make it a bit more ergonomic?

File: src/pages/journal/journal.ts [hunk 2]
```diff
@@ -32,10 +32,11 @@ export class JournalPage extends BasePageComponent implements OnInit, OnDestroy
...
 
-
   pageState: JournalPageState;
 
   loadingSpinner: Loading;
+  pageRefresher: Refresher;
+
   toast: Toast;
   subscription: Subscription;
   start = '2018-12-10T08:10:00+00:00';
```
Comment (ammarhaiderbjss) [inline src/pages/journal/journal.ts]: Is this still needed? Can we remove if not?

Comment (inhand) [reply src/pages/journal/journal.ts]: Yes its needed so we have a reference to it and clear it when the journal has stopped loading

File: src/pages/journal/journal.html (hunk 1 above)
Comment (inhand) [reply src/pages/journal/journal.html]: Yes we are now clearing out the previously dynamically generated slots

Comment (inhand) [reply src/pages/journal/journal.html]: Yep can have a look at increasing the pull

**Pull request merged**